packages/runtime/src/types/consumption/IdentityMetadataDTO.ts

```diff
@@ -0,0 +1,5 @@
+export interface IdentityMetadataDTO {
+    reference: string;
+    key: string | undefined;
```

Comment:
use `key?: string`

Reply:
I think I had a reason not to do that, but as I don't know the reason anymore ok